src/redist/targets/packaging/osx/clisdk/resources/cs.lproj/conclusion.html

```diff
@@ -5,7 +5,7 @@
     </head>
     <body>
         <div align="left" style="font-family: Helvetica;padding-left:10px">
-            <p>The following was installed</h2>
+            <p id="installlist">The following was installed</h2>
```

Comment:
1. eng/common/generate-locproject.ps1 won't currently find HTML files though @jonfortescue or I should be able to add support
2. Recommend making changes that are _inputs_ to the loc process only in the original (English) file. The handback PRs will copy that file and begin translation from that point automatically.
3. Might need a loc fix because `cs.lproj` is an unusual path segment. /cc @cristianosuzuki77 and @jeeyyoo

Reply:
1. Should I hold off on making the id changes until html support exists and how can I get a short-term translation for these?
2. For this, are you just saying in the future to only update the english version?  For this PR, I assume I would need ids in each of the loc files too.
3. Let me know.

Reply:
1. One option is to update the generate-locproject.ps1 in this PR and get a similar update into Arcade between now and when the next Arcade update comes. Thoughts @jonfortescue❔
2. I bet the ids are inputs and that it's up to the loc team whether they appear in the output files or not. @cristianosuzuki77❔
3. @jonfortescue I bet we can use an existing clone language set. The HTML file addition would also need `CloneLanguageSet = "Outlook_iOS_CloneLanguages"`. `"IOSTranslator_CloneLanguages"` or `"iOS_Apex43_CloneLanguages"` may be a better match; I didn't check all folder names in clisdk/resources/ to confirm which works best.